system/mp/mpcomm.cpp

```diff
@@ -1889,6 +1914,14 @@ int CMPConnectThread::run()
         }
         if (sock) {
             try {
+                if (soMaxCheck)
+                {
+                    if (numMPAccepts++ >= soMaxConn)
```

Comment:
Could also maybe check other side for connect() failures and give warning, but that should be a separate task.
Ultimately should launch or pass to another thread to complete the MP handshake so that accept()s happen very quickly to reduce chance of backlog growing large.